yang/vyatta-protocols-frr-zebra-v1.yang

```diff
@@ -0,0 +1,227 @@
+module vyatta-protocols-frr-zebra-v1 {
+  namespace "urn:vyatta.com:mgmt:vyatta-protocols-frr-zebra:1";
+  prefix vyatta-protocols-frr-zebra-v1;
+
+  import vyatta-protocols-v1 {
```

Comment:
This doesn't seem to be used.

Reply:
I have removed the unused imports i.e vyatta-protocols-v1